rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigsForTest.java

```diff
@@ -160,6 +160,17 @@ private static List<ConsensusRule> getHop401Rskips() {
         return rskips;
     }
 
+    private static List<ConsensusRule> getFingerroot500Rskips() {
+        List<ConsensusRule> rskips = new ArrayList<>();
+        rskips.addAll(Arrays.asList(
+            ConsensusRule.RSKIP374,
+            ConsensusRule.RSKIP375,
+            ConsensusRule.RSKIP377
```

Comment:
Are those all the RSKIPs we have in fingerroot?

Reply:
Done